00006_array_sum.js

```diff
@@ -0,0 +1,14 @@
+// 6. Find the sum of elements in a given array. Array should be an input to the program
+
+let inputArray = [];
+
+const sumOfArrayElements = (inputArray) => {
+    let result = 0;
+    for (let i = 0; i < inputArray.length; i++) {
+        result = result + inputArray[i];
+    }
+    return result
+}
+
+sumOfArrayElements(inputArray);
```

Comment:
Why you need this?

Reply:
Actually I was meant to express how the function is being called and once I sawed your were exporting the program to check. So I thought if you were using each code, this would be useful. I can remove this from every program and I can update that.